styleframe/tests/style_frame_tests.py

```diff
@@ -1,5 +1,6 @@
 import os
 import unittest
+import inspect
```

Comment:
Let's keep the convention of alphabetically sorting the imports.

```suggestion
import inspect
import os
import unittest
```